package matching;

message Compound {
  required string parsed_name = 1;
  optional int32 parsed_coeff = 2 [default = 1];
  
  repeated string match_names = 3;
}

message TestReaction {
  required string query = 1;
  
  repeated Compound reactants = 2;
  repeated Compound products = 3;
}

message TestReactions {
  repeated TestReaction reactions = 1;
}